### ISSUE TITLE
Add term osteosclerosis of the femur

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10489]
**Preferred term label:**
Osteosclerosis of the femur

**Synonyms**
Sclerotic femur

**Definition (free text, please give PubMed ID)**
An abnormal increase of bone mineral density of the femur. Sclerosis is normally detected on a radiograph as an area of increased opacity. [based on Sclerosis of foot bone HP:0100925] PMID: 15326622

**Parent term (use hpo.jax.org/app)**
Increased bone mineral density HP:0011001

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Camurati-Engelmann disease (omim131300) PMID: 15326622

**Your nano-attribution (ORCID)**
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@lisannesw 
@azankl
Adding new term: 
- ID: HP:6000708
- label: Femoral osteosclerosis
- parents: [Increased bone mineral density (HP:0011001)](https://hpo.jax.org/app/browse/term/HP:0011001) | [Abnormal femur morphology (HP:0002823)](https://hpo.jax.org/app/browse/term/HP:0002823)
- pmid: PMID:15326622
- def: An abnormal increase of bone mineral density of the femur. Sclerosis is normally detected on a radiograph as an area of increased opacity.
- comment: 
- orcid: orcid:0009-0007-1020-5412
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10489
- synonyms: Osteosclerosis of the femur(NONE)
